modules/phase_field/src/postprocessors/GrainTracker.C

```diff
@@ -1369,7 +1378,7 @@ GrainTracker::updateFieldInfo()
       {
         const Elem * elem = mesh.elem(entity);
         std::vector<Point> centroid(1, elem->centroid());
-        if (_ebsd_reader)
+        if (_ebsd_reader && _first_time)
```

Comment:
@tonkmr, @frombs - I was missing this little guy so the `UNIQUE_REGION` and `VARIABLE_COLORING` was unstable for EBSD runs. This PR should fix that. Additionally the halo level was only 1 which isn't thick enough to avoid coalescence so that's two problems.

Now the GrainTracker seems to be working a lot better with EBSD data with this patch. However there's now a new issue that's occuring on the example problem. There's a "split" grain on the left hand side of the boundary that's just close enough that the grain tracker sees it as a single grain in the initial dataset. On the second time step the two pieces have shrunk enough so that the GrainTracker thinks a new grain is forming there. 

![ebsd_data](https://cloud.githubusercontent.com/assets/3036960/18369425/e51b2ba2-75e2-11e6-892b-112f52b65482.png)
 
That's just fine from the GrainTracker point of view but it's raising havoc with the RGB object. It "should" be possible to detect this case and avoid creating a new grain in this case but I'll need to work on that. 

@frombs - Grab this branch and try your dataset again. Sorry for the constant changes but these are tough problems!